src/blogtoblog/BlogToBlogImporter.ts

```diff
@@ -138,27 +137,34 @@ export default class BlogToBlogImporter extends PageImporter {
     let category;
     if (topicsArr.length) {
       category = topicsArr.shift();
-      topics = topicsArr.join(', ');
       const categoryRow = document.createElement('tr');
       table.append(categoryRow);
       const categoryTitle = document.createElement('td');
       categoryTitle.textContent = 'Category';
       categoryRow.append(categoryTitle);
       const categoryData = document.createElement('td');
-      categoryData.textContent = category;
+      categoryData.textContent = category.trim();
```

Comment:
Nitpicking: Why adding the `trim()` here and not when setting the `category` value ? There is already a `shift()` there